modules/test/playwright/tests/portal-workflow-kaleo-designer-web/startNode.spec.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {apiHelpersTest} from '../../fixtures/apiHelpersTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {workflowPagesTest} from '../../fixtures/workflowPagesTest';
+import {getRandomInt} from '../../utils/getRandomInt';
+
+export const test = mergeTests(apiHelpersTest, loginTest(), workflowPagesTest);
+
+let workflowDefinitionId: number;
+let workflowDefinitionName: string;
+
+test.beforeEach(async ({apiHelpers}) => {
+	const singleApproverWorkflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.getWorkflowDefinitionByName(
+			'Single Approver'
+		);
+
+	workflowDefinitionName = 'Copy of Single Approver' + getRandomInt();
+
+	const workflowDefinition =
+		await apiHelpers.headlessAdminWorkflow.postWorkflowDefinitionSave(
+			workflowDefinitionName,
+			singleApproverWorkflowDefinition
+		);
+
+	workflowDefinitionId = workflowDefinition.id;
+});
+
+test.afterEach(async ({apiHelpers}) => {
+	await apiHelpers.headlessAdminWorkflow.deleteWorkflowDefinition(
+		workflowDefinitionId
+	);
+});
+
+test('can add and edit node into diagram by drag&drop', async ({
```

Comment:
Hey @larissacribeiro, could you explaim me why you decide use `drag&drop` instead `drag and drop`.

Reply:
hey Igor, no particular reason, this is the way it was written on the former poshi test as well as on the other that go through the same action, so I didn't think of changing it. But I can do it, no problem!

Reply:
No problem @larissacribeiro, it really was a question. If we already have other examples, I believe we can continue in this way.